docs/snippets/defaults_genai.mdx

```diff
@@ -0,0 +1,12 @@
+```
```

Comment:
where is this snippet used?

Reply:
It's not, but figured we'd want it when we do the model later? Wasn't sure if we'd do an AI variant of calculations.mdx, so really just here as a placeholder.